### ISSUE TITLE
V13.1.1 Feedback on mismatched encoding

### ISSUE THREAD

**Issue by vanderaj** [OWASP/ASVS#513]
Jim, this one doesn’t seem important... except that I went to an @orange_8361  talk at Black Hat a few years ago on abusing this very mechanism for fun and profit. I think we need to rephrase this to make it clearer as to why this is important. 

Additionally, I believe this to be CWE-175 ‪‪not 116.

**Comment by vanderaj** [comment]
Basically, I would like a reference to ‪ 
https://www.blackhat.com/docs/us-17/thursday/us-17-Tsai-A-New-Era-Of-SSRF-Exploiting-URL-Parser-In-Trending-Programming-Languages.pdf

**Comment by jmanico** [comment]
Ok that talk just freaked me out, I see why this is important. Damn damn parsers!

**Comment by vanderaj** [comment]
Best. Talk. Of. Black. Hat. 2017. Period.
